lib/quadquizaminos_web/controller/session_controller.ex

```diff
@@ -0,0 +1,33 @@
+defmodule QuadquizaminosWeb.SessionController do
+  use QuadquizaminosWeb, :controller
+  alias Quadquizaminos.Accounts
+
+  def new(conn, _params) do
+    render(conn, "new.html")
+  end
+
+  def create(conn, %{"session" => %{"name" => name}}) do
+    case Accounts.find_or_create(name) do
+      {:ok, user} ->
+        conn
+        |> put_flash(:info, "Successfully authenticated.")
+        |> assign(:current_user, user)
+        |> put_session(:uid, user.name)
```

Comment:
The UID should be the  user.uid, not the user name. If this means that we creat a uid same as the name then thats okay

Reply:
worked on it